src/V3Width.cpp

```diff
@@ -3298,6 +3298,26 @@ class WidthVisitor final : public AstNVisitor {
                     relinkHandle.relink(patp);
                 }
             }
+
+            std::vector<std::pair<AstNodeDType*, AstPatMember*>> type_defaults;
+            for (AstPatMember* patp = VN_CAST(nodep->itemsp(), PatMember); patp;
+                 ) {
+                if (patp->isTypedDefault()) {
+                    AstNodeDType* type = VN_CAST(patp->keyp(), NodeDType);
+                    auto matching_type_it
+                        = find_if(type_defaults.begin(), type_defaults.end(),
+                                  [&type](auto x) { return type->similarDType(x.first); });
+                    if (matching_type_it == type_defaults.end()) {
+                        type_defaults.push_back({type, patp});
+                    }
+                    auto helper = VN_CAST(patp->nextp(), PatMember);
+                    patp->unlinkFrBack();
```

Comment:
Does this need to be deleted if it wasn't push_backed?

Reply:
Yeah, you are right, it doesn't need to be deleted. Just need to skip over it in the next phase... Fixed